build.gradle

```diff
@@ -157,6 +157,15 @@ javadoc {
 	options.addStringOption('Xdoclint:none', '-quiet')
 }
 
+task releaseJavadoc(type: Javadoc) {
```

Comment:
This should consider the excludes as we mentioned before, can it be done?

Reply:
yes, i think. but i was hoping we could come up with an "includes" list instead.

Reply:
tabled for a future PR